cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -18,157 +18,261 @@ var Notification = require( '../molecules/Notification' );
  */
 function FilterableListControls( element ) {
   var BASE_CLASS = 'o-filterable-list-controls';
-
   var _dom = atomicCheckers.validateDomElement(
     element, BASE_CLASS, 'FilterableListControls' );
   var _form = _dom.querySelector( 'form' );
   var _notification;
-  var _fields = [];
+  var _fieldGroups;
 
-  var _validatorDefaults = {
-    inputs: [
-      'input',
-      'textarea',
-      'select'
+  var _defaults = {
+    ignoreFieldTypes: [
+      'hidden',
+      'button',
+      'submit',
+      'reset',
+      'fieldset'
+    ],
+    groupFieldTypes: [
+      'radio',
+      'checkbox'
     ]
   };
 
+  var _validators = {
+    date: function( field, currentStatus ) {
+      var status = currentStatus || {};
+      var dateRegex =
+        /^\d{2}$|^\d{4}$|^\d{2}\/(?:\d{4}|\d{2})$|^\d{2}\/\d{2}\/\d{4}$/;
+      if ( field.value && dateRegex.test( field.value ) === false ) {
+        status.msg += ERROR_MESSAGES.DATE.INVALID;
```

Comment:
yeah, the messages are getting concatenated  in https://github.com/cfpb/cfgov-refresh/pull/1451/files#diff-373a9942243971a4ddce967a7dd02b20R137